.env

```diff
@@ -18,10 +18,11 @@ MOODLE_ADMIN_PASS=Passw0rd!
 MOODLE_DBTYPE=pgsql
 MOODLE_FULL_NAME=SC Moodle Training
 MOODLE_SHORT_NAME=SCMT
+MOODLE_UPGRADE_KEY=Upgr@deM3!
 
 # Database variables
 PGSQL_DATABASE=moodle
 PGSQL_USER=moodle
-PGSQL_PASSWORD=password123
+PGSQL_PASSWORD=P0stGre$QL!
```

Comment:
Maybe we should make the default 'changeme' or something to indicate it should be modified.

Reply:
Yeah, good idea.

Reply:
Done